feature-group-definitions/blocking-render.yml

```diff
@@ -0,0 +1,11 @@
+name: 'blocking="render"'
+details: "The `blocking="render"` attribute for `<link>`, `<script>`, and `<style>` elements blocks rendering until the external script or stylesheet has been loaded."
```

Comment:
@domenic given that you reviewed https://github.com/whatwg/html/pull/7474 you must understand this feature deeply. I made an attempt at describing what this features does. I know it must be wrong because it's [potentially render-blocking](https://html.spec.whatwg.org/multipage/urls-and-fetching.html#potentially-render-blocking) and not "definitely render-blocking", but I don't know when it doesn't actually block, or whether developers need to know about this from a short description.

Advice appreciated :)

Reply:
I think your summary seems reasonable. The caveats are mostly detailed in https://html.spec.whatwg.org/#render-blocking-mechanism . Things like:

- This only works for elements in the `<head>`
- There's an implementation-defined timeout
- Only works for text/html documents

... that sort of thing.

@noamr is probably our current expert on this?

Reply:
- Works only in the head
- Doesn't work on non-async classic scripts
- For links, only wokrs on `<link rel=stylesheet>` and `<link rel=expect`>. For the latter it would block until the element indicated by `href` is seen and is fully parsed (end tag seen or added programmatically).

Reply:
Thanks @domenic and @noamr for the feedback. Of those caveats, I think the implementation-defined timeout is the most relevant to understand, and what some (evil QA people) might wonder about when reading about the feature. Is the timeout the same in all browsers, and what is it?

I'll add some comments to the YAML for things you've listed but I'm not sure should go in the description, so it's easier to find for future web-features maintainers.

Reply:
> Thanks @domenic and @noamr for the feedback. Of those caveats, I think the implementation-defined timeout is the most relevant to understand, and what some (evil QA people) might wonder about when reading about the feature. Is the timeout the same in all browsers, and what is it?
> 

Chrome currently doesn't implement  such timeout. Since render blocking always relies on resource loading (js, css, html), the usual network timeouts apply.
> 
> 
> I'll add some comments to the YAML for things you've listed but I'm not sure should go in the description, so it's easier to find for future web-features maintainers.